com.unicornsonlsd.finamp.yaml

```diff
@@ -11,37 +11,35 @@ finish-args:
-      - install -Dm644 icons/48x48/apps/finamp.png /app/share/icons/hicolor/48x48/apps/com.unicornsonlsd.Finamp.png
-      - install -Dm644 icons/64x64/apps/finamp.png /app/share/icons/hicolor/64x64/apps/com.unicornsonlsd.Finamp.png
-      - install -Dm644 icons/128x128/apps/finamp.png /app/share/icons/hicolor/128x128/apps/com.unicornsonlsd.Finamp.png
-      - install -Dm644 icons/256x256/apps/finamp.png /app/share/icons/hicolor/256x256/apps/com.unicornsonlsd.Finamp.png
-      - install -Dm644 icons/384x384/apps/finamp.png /app/share/icons/hicolor/384x384/apps/com.unicornsonlsd.Finamp.png
-      - install -Dm644 icons/512x512/apps/finamp.png /app/share/icons/hicolor/512x512/apps/com.unicornsonlsd.Finamp.png
+      - install -Dm644 icons/16x16/apps/finamp.png /app/share/icons/hicolor/16x16/apps/com.unicornsonlsd.finamp.png
+      - install -Dm644 icons/32x32/apps/finamp.png /app/share/icons/hicolor/32x32/apps/com.unicornsonlsd.finamp.png
+      - install -Dm644 icons/48x48/apps/finamp.png /app/share/icons/hicolor/48x48/apps/com.unicornsonlsd.finamp.png
+      - install -Dm644 icons/64x64/apps/finamp.png /app/share/icons/hicolor/64x64/apps/com.unicornsonlsd.finamp.png
+      - install -Dm644 icons/128x128/apps/finamp.png /app/share/icons/hicolor/128x128/apps/com.unicornsonlsd.finamp.png
+      - install -Dm644 icons/256x256/apps/finamp.png /app/share/icons/hicolor/256x256/apps/com.unicornsonlsd.finamp.png
+      - install -Dm644 icons/384x384/apps/finamp.png /app/share/icons/hicolor/384x384/apps/com.unicornsonlsd.finamp.png
+      - install -Dm644 icons/512x512/apps/finamp.png /app/share/icons/hicolor/512x512/apps/com.unicornsonlsd.finamp.png
       - mkdir -p /app/bin
       - ln -s /app/finamp /app/bin/finamp
-      - sed -i 's/Icon=finamp/Icon=com.unicornsonlsd.Finamp/' finamp.desktop.m4
+      - sed -i 's/Icon=finamp/Icon=com.unicornsonlsd.finamp/' finamp.desktop.m4
       - sed -i 's/__INSTALL_PATH__\/finamp/finamp/' finamp.desktop.m4
-      - m4 finamp.desktop.m4 > com.unicornsonlsd.Finamp.desktop
-      - install -Dm644 com.unicornsonlsd.Finamp.desktop /app/share/applications/com.unicornsonlsd.Finamp.desktop
-      - install -Dm644 com.unicornsonlsd.Finamp.metainfo.xml /app/share/appdata/com.unicornsonlsd.Finamp.metainfo.xml
+      - m4 finamp.desktop.m4 > com.unicornsonlsd.finamp.desktop
+      - install -Dm644 com.unicornsonlsd.finamp.desktop /app/share/applications/com.unicornsonlsd.finamp.desktop
+      - install -Dm644 com.unicornsonlsd.finamp.metainfo.xml /app/share/appdata/com.unicornsonlsd.finamp.metainfo.xml
     sources:
       - type: archive
         only-arches:
           - x86_64
         url: https://github.com/jmshrv/finamp/releases/download/0.9.7-beta/finamp-0.9.7-beta-linux-release.tar.gz
-        sha256: 3fc54090141663bdeb0c378766c346d527c3f27a2a4c61a6053174dfc1f738f4
+        sha256: 9916d7df3fd83b17391b1a66c4ad2dd86542b830cd837e86c59d00e447ce3086
```

Comment:
retagging is bad. this should have been a new version / tarball.

Reply:
So a new upstream release is needed?

Reply:
it's a bad practice in general. because then there is no way to distinguish. But it's too late now.

Reply:
There was no re-tagging, just the attached assets were updated. Those assets aren't officially released yet, we're in the process of making a soft-release. The official release will happen with a properly tagged and afterwards static release, like it should.  
Creating a new release tag for testing/debugging purposes doesn't seem like a better practice either ^^

Reply:
it's the same thing. Same tarball name, content changed mean it's a different release. It's bad practice overall.